aw_client/client.py

```diff
@@ -226,6 +226,18 @@ def delete_bucket(self, bucket_id: str):
     def setup_bucket(self, bucket_id: str, event_type: str):
         self.create_bucket(bucket_id, event_type, queued=True)
 
+    # Import & export
+
+    def export_all(self) -> dict:
+        return self._get('export').json()
+
+    def export_bucket(self, bucket_id) -> dict:
+        return self._get('buckets/{}/export'.format(bucket_id)).json()
+
+    def import_bucket(self, bucket: dict) -> None:
+        endpoint = "import"
+        self._post(endpoint, {"buckets": {bucket["id"]: bucket}})
```

Comment:
Can you add an test for this?

Reply:
We don't have extensive tests for the other endpoints, and I'm not going to set that up, so no.

Maybe there is in the aw-server repo? We should put them all in one place somehow so we can do proper integration testing for both `aw-server-python` and `aw-server-rust` with a single test suite.